doc/changes/0.10.2.rst

```diff
@@ -85,6 +85,8 @@ Enhancements
 
 - :bdg-success:`API` Improve ``contrasts`` allowing fixed effects on F contrasts (:gh:`3203` by `Bertrand Thirion`_)
 
+- :bdg-dark:`Code` ``nilearn/tests/test_signal.py`` Remove unnecessary code in test (:gh:`4175` by `Paul Reiners`_).
```

Comment:
Make sure you also add your name in the citation.cff file: otherwise the documentation won't find the link associated to your name.

check this section of the contributing guidelines.
https://nilearn.github.io/stable/development.html#changelog

Reply:
See the failure of the doc build: https://github.com/nilearn/nilearn/actions/runs/7485429117/job/20376192765?pr=4208#step:34:589

Reply:
Added my name to the citation.cff file.